pkg/sidecar/config/config.go

```diff
@@ -206,6 +214,18 @@ func (s *ScyllaConfig) setupEntrypoint(ctx context.Context) (*exec.Cmd, error) {
 	return scyllaCmd, nil
 }
 
+func (s *ScyllaConfig) validateCpuSet(ctx context.Context, cpusAllowed string, shards int) error {
```

Comment:
How about a plain function?
Please add a comment.

Reply:
Sure but I wanted access to the logger for that information log about less than optimal setup. It's not a problem to pass it in though.